src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -185,5 +185,36 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
 
 		snapshot("screenshot_certificate_overview_vaccination_and_test_certificate")
 	}
+	
+	func test_screenshot_MultipleFamilyTestCertificates() throws {
```

Comment:
you have to also check this particular UI test in screenshot test plan otherwise the screenshots will not be obtained.

Reply:
Added